plugins/airbrake/src/api/production-api.test.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ProductionAirbrakeApi } from './production-api';
+import nock from 'nock';
```

Comment:
Prefer using `msw` to mock network requests.

You can read more here: https://backstage.io/docs/architecture-decisions/adrs-adr007

Reply:
Gotcha, using `msw` now :)